src/randoop/operation/NonreceiverTerm.java

```diff
@@ -284,14 +307,16 @@ public static NonreceiverTerm parse(String s) throws OperationParseException {
       typeString = "java.lang.String";
     }
     
-    Class<?> type = Reflection.classForName(typeString, true);
-    if (type == null) {
+    Class<?> type;
+    try {
+      type = TypeNames.getTypeForName(typeString);
+    } catch (ClassNotFoundException e1) {
       String msg = "Error when parsing type/value pair " + s + ". A primitive value declaration description must be of the form "
-        + "<type>:<value>" + " but the <type> given (\"" + typeString + "\") was unrecognized.";
-      throw new OperationParseException(msg);
+          + "<type>:<value>" + " but the <type> given (\"" + typeString + "\") was unrecognized.";
+        throw new OperationParseException(msg);
     }
+    
     Object value = null;
```

Comment:
Can you remove the initialization?  The rhs there suggests that value might not be set; leaving it off makes clear that value is always set.